openedx_certificates/processors.py

```diff
@@ -0,0 +1,224 @@
+"""
+This module contains processors for certificate criteria.
+
+The functions prefixed with `retrieve_` are automatically detected by the admin page and are used to retrieve the
+IDs of the users that meet the criteria for the certificate type.
+
+We will move this module to an external repository (a plugin).
```

Comment:
I would suggest just putting this into a `plugins/completion/processor.py` file in the same repo, along with the corresponding generators (for "Certificate of completion" & "Badge of achievement/completion")